src/tests/askFourEarning.ts

```diff
@@ -0,0 +1,46 @@
+import { Test } from '../lib/variants';
+import { cacheAsync } from '../lib/cache';
+import { fetchDefaultEpicContent } from '../api/contributionsApi';
+
+export const askFourEarningHardcodedTest = async (): Promise<Test> => {
+    const fiveMinutes = 60 * 5;
+    const [, fetchDefaultEpicContentCached] = cacheAsync(fetchDefaultEpicContent, fiveMinutes);
```

Comment:
I _think_ that because of the way `cacheAsync` is implemented we'll build a new cache for each invocation of `askFourEarningHardcodedTest` so won't get the benefit of the caching. I reckon if you pull this line up out of the `askFourEarningHardcodedTest` function body that would have the desired effect. (Though would be good to verify that before merging this, I might be wrong 😸).

Reply:
Yeah exactly. 👍

Reply:
See the uses in `server.tsx` for examples.

Reply:
Cool, thanks for the reminder!